bowling.js

```diff
@@ -0,0 +1,47 @@
+class Bowling {
+  constructor() {
+    this.rolls = [];
+  }
+
+  roll(pins) {
+    this.rolls.push(pins);
+  }
+
+  _isSpare(frameScore) {
+    return frameScore === 10;
+  }
+
+  _isStrike(rollIndex) {
+    return this.rolls[rollIndex] === 10;
+  }
```

Comment:
This is a detail, but these two methods have name that make it look like they do very similar things, however they expect a very different argument. I wonder if there's a way to either:
- use the same kind of argument (eg the rollIndex of the first roll in frame, maybe?) 
- or have names that hint at their difference.

This comment is about code design / programmer experience of the code. When programming, we rely on many small hints to quickly understand what the code does / how it works. names are a big one, obviously, and here you use a number of conventions really well (the `_`, the `is` hinting at a boolean return), but consistency of method usage, or even where it is defined in the class are other hints a programmers considers - even unconsciously - when choosing which method to use.